UIKitVK/UIKitVK/Model/User/User.swift

```diff
@@ -6,15 +6,19 @@ import RealmSwift
 
 // Пользователь
 final class User: Object, Codable {
-    @objc dynamic var id: Int
-    @objc dynamic var firstName: String
-    @objc dynamic var lastName: String
-    @objc dynamic var friendAvatar: String?
+    @Persisted var id: Int
+    @Persisted var firstName: String
+    @Persisted var lastName: String
+    @Persisted var friendAvatar: String?
 
     private enum CodingKeys: String, CodingKey {
         case id
         case firstName = "first_name"
         case lastName = "last_name"
         case friendAvatar = "photo_100"
     }
+
+    override static func primaryKey() -> String? {
+        "id"
+    }
```

Comment:
```suggestion

    override static func primaryKey() -> String? {
        "id"
    }
```

тоже самое сделай через @Persisted(primaryKey: true)